.github/workflows/e2e-tests.yml

```
@@ -0,0 +1,54 @@
+        node-version: '20'
+        cache: 'npm'
+    
+    - name: Install dependencies
+      run: npm ci
+    
+    - name: Install Playwright Browsers
+      run: npx playwright install --with-deps
+    
+    - name: Build application
+      run: npm run build
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)